src/pages/developing/frameworks/react.mdx

```diff
@@ -60,17 +60,21 @@ behavior in prototype and production work.
 <Title> Using npm: </Title>
 
 ```bash
-npm install --save @carbon/react
+npm install --save @carbon/react react react-dom
```

Comment:
I'd wager by the time someone installs `@carbon/react` they're going to have already set up some kind of app/framework that will either install react and react-dom for them or instruct them to do so. I think the reminder note below is enough. Also they're going to get warnings about these in their console anyway if they don't have them installed for some reason.


```suggestion
npm install --save @carbon/react
```